spec/unit/multi/register_spec.rb

```diff
@@ -35,4 +35,23 @@
     expect(bars.total).to eq(15)
     expect(bars.current).to eq(0)
   end
+
+  it "updates total based on children totals" do
+    bars = TTY::ProgressBar::Multi.new("main [:bar]", output: output)
+
+    bars.register("[:bar]", total: 1)
+    expect(bars.total).to eq(1)
+
+    bars.register("[:bar]", total: 1)
+    expect(bars.total).to eq(2)
+  end
+
+  # https://github.com/piotrmurach/tty-progressbar/issues/49
```

Comment:
I tend to leave comments like this out. The test subject expresses the issue/solution well enough for me.